### PR TITLE
Rewrite run.sh script to allow running tests locally

### PR DESCRIPTION
**Pull request by tsusanka**
Change run-travis.sh to universal run.sh that can be run locally. Uses new trezor-user-env. Things to do:

- [ ] Check on Mac. I don't have one at the moment. Should work also with GUI if `-e DISPLAY=:0` is replaced with `-e DISPLAY=docker.for.mac.host.internal:0`.
- [ ] Check if Travis works. For some reason [it does not run](https://travis-ci.org/github/trezor/connect/builds/743466649) now but develop does not run either so it is probably not related to this PR.

### CODE REVIEW

File: tests/run.sh [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  echo ""
+  echo "Options:"
+  echo "  -g       Run tests with emulator graphical output"
+  echo "  -f       Use specific firmware version, for example: 2.1.4., 2.3.0"
+  echo "  -i       Included methods only, for example: applySettings,signTransaction"
+  echo "  -e       All methods except excluded, for example: applySettings,signTransaction"
+  echo "  -c       Collect coverage"
+}
+
+FIRMWARE='2.3.2'
```
Comment (szymonlesisz) [inline tests/run.sh]: 2.3.4

File: tests/run.sh [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    # TODO: I don't think these options are needed:
+    # -v /tmp/.X11-unix:/tmp/.X11-unix:rw \
+    # -e QT_X11_NO_MITSHM=1 \
+
+    echo "Running docker container with a GUI support with ID:"
+    echo "$id"
+  fi
+
+  echo "Waiting for the trezor-user-env to load up"
+  sleep 10
```
Comment (szymonlesisz) [inline tests/run.sh]: not sure about that delay. it may cause some unexpected race conditions. isn't there other way to catch trezor-user-env load event?

**Review by szymonlesisz — COMMENTED** [review]
rebase this branch on top of https://github.com/trezor/connect/pull/708
failing test are fixed there.

i've tried it on mac, but its failing on websocket connecton. im blaming docker process command, because if i run it as standalone the output is:
```
> docker run -it -e SDL_VIDEODRIVER="dummy" -p "21326:21326" -p "9001:9001" -p "21325:21325" registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env:latest /trezor-user-env/run.sh
> Python 3.8.5
> trezorctl, version 0.12.2
> Starting trezor-user-env server
>
```

File: tests/run.sh (hunk 1 above)
Comment (tsusanka) [reply tests/run.sh]: f7dc340

File: tests/run.sh (hunk 2 above)
Comment (tsusanka) [reply tests/run.sh]: Yeah probably yes, I was just lazy. I'll have a look.

Comment (tsusanka) [reply tests/run.sh]: https://github.com/trezor/connect/pull/709/commits/0e2c2830795d3bd75ee6c4fcb9c38b4be6bd4999

**Comment by tsusanka** [comment]
Please test https://github.com/trezor/connect/pull/709/commits/0e2c2830795d3bd75ee6c4fcb9c38b4be6bd4999 on Mac and if it works I can squash and we'll merge.

**Review by szymonlesisz — APPROVED** [review]
it's failing on zcash tests as expected, they are fixed in other branch

**Pull request merged**